activestorage/test/models/serialize_attached_test.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "database/setup"
+
+class ActiveStorage::AttachmentsTest < ActiveSupport::TestCase
+  include ActiveJob::TestHelper
+
+  setup do
+    @user = User.create!(name: "Josh")
+  end
+
+  teardown { ActiveStorage::Blob.all.each(&:purge) }
+
+  test "serializing a object without attachments" do
+    # the serialize_attachments method was used to indicate that :avatar and :cover_photo will be included in the serialization
+    assert_equal true, @user.as_json.has_key?("avatar_attached")
```

Comment:
Do you think it's better to assert the expected value instead of just assert that the key exists?

Reply:
I agree with you